modules/packages/LinearAlgebra.chpl

```diff
@@ -823,6 +825,11 @@ proc matPow(A: [], b) where isNumeric(b) {
   if !isSquare(A) then
     halt("Array not square");
 
+  if (b < 0) {
```

Comment:
This if should check for `usingLapack && b<0`.

This way if we are not using lapack, we'll still call `_expBySquaring` which'll halt if b<0.